### DISCUSSION TITLE
Display Rotate 90°

### DISCUSSION THREAD

**Discussion by mefiu1** [KlipperScreen/KlipperScreen#127] [Ideas]
Is there a solution to rotate a touch display by 90°? If not could you add this as Future ?

Greetings

**Comment by HermanGaviriaMarin** [comment]
I was looking for this same solution. Turns out that it should be done as per the manufacturers directions, that is, find the official website of documentation of your specific screen and it is very possible that the manufacturer has some SSH commands you can run on putty to rotate or flip the screen. You maybe have to change klipper screen to make sure that touch input registers but thats a maybe. 

A quick example of what I mean is here on my displays Github:

https://github.com/goodtft/LCD-show#:~:text=How%20to%20rotate,the%20latest%20drivers)

here they explain two ways flipping it, but remember that these directions will probably not work for you unless you have the same screen as I do, which is the 
https://www.amazon.com/Elecrow-800x480-Interface-Supports-Raspberry/dp/B013JECYF2/ref=sr_1_1_sspa?keywords=5+inch+tft&qid=1639335478&sr=8-1-spons&psc=1&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEzSTY2RUlPV1lFMEhNJmVuY3J5cHRlZElkPUEwMjUzMTE2V1hMVDJGWDA5UEFaJmVuY3J5cHRlZEFkSWQ9QTAwNDQ5NDVTWEJLSDgwN1ZZTFUmd2lkZ2V0TmFtZT1zcF9hdGYmYWN0aW9uPWNsaWNrUmVkaXJlY3QmZG9Ob3RMb2dDbGljaz10cnVl,

**Comment by alfrix** [comment]
display rotation is handled by the OS, the docs has been updated with multiple examples to aid users in achieving rotation https://klipperscreen.readthedocs.io/en/latest/Hardware/#screen-rotation
